src/AudioFileSDLSound.cpp

```diff
@@ -13,6 +13,7 @@
 #include <memory>
 #include <mutex>
 #include <vector>
+#include <stdexcept>
```

Comment:
```suggestion
#include <stdexcept>
#include <vector>
```
(just for alphabetical order)